jobs/snort/templates/rules/local.rules.erb

```diff
@@ -1 +1,3 @@
-<%= p('snort.rules', '') %>
+<% p("snort.rules", []).each do |rule| %>
+  <%= rule %>
```

Comment:
I can't believe I'm commenting on a 3 line PR, but is snort OK with a leading space or tab in the rules list (if not should probably nuke this)